uk.co.compucorp.civicrm.hrcore/gulp/tasks/sass.js

```diff
@@ -0,0 +1,84 @@
+var bulk = require('gulp-sass-bulk-import');
+var civicrmScssRoot = require('civicrm-scssroot')();
+var colors = require('ansi-colors');
+var gulp = require('gulp');
+var gulpSequence = require('gulp-sequence');
+var path = require('path');
+var sass = require('gulp-sass');
+var stripCssComments = require('gulp-strip-css-comments');
+
+var utils = require('../utils');
+
+module.exports = [
+  {
+    name: 'sass',
+    fn: function (cb) {
+      var sequence;
+
+      if (utils.canCurrentExtensionRun('sass')) {
+        sequence = utils.addExtensionCustomTasksToSequence([
+          utils.spawnTaskForExtension('sass:sync', syncTask),
+          utils.spawnTaskForExtension('sass:main', mainTask)
+        ], 'sass');
+
+        gulpSequence.apply(null, sequence)(cb);
+      } else {
+        console.log(colors.yellow('Not eligible for this task, skipping...'));
```

Comment:
I am not sure we want it yellow. Do we?

Reply:
yellow because it's warning, not an error

Reply:
I didn't mean error, I meant `info`.